Source/Core/VideoCommon/HiresTextures.cpp

```diff
@@ -243,10 +243,11 @@ std::string HiresTexture::GenBaseName(const u8* texture, size_t texture_size, co
     }
     break;
   case 16384 * 2:
-    for (size_t i = 0; i < texture_size / 2; i++)
+    for (size_t i = 0; i < texture_size / sizeof(u16); i += sizeof(u16))
```

Comment:
Since `i` is now an offset and not an index into a `u16` array anymore, should `i < texture_size / sizeof(u16)` be `i < texture_size` instead?